app/post.go

```diff
@@ -955,7 +950,11 @@ func (a *App) SearchPostsInTeamForUser(terms string, userId string, teamId strin
 }
 
 func (a *App) GetFileInfosForPostWithMigration(postId string) ([]*model.FileInfo, *model.AppError) {
-	pchan := a.Srv.Store.Post().GetSingle(postId)
+	post, errPost := a.Srv.Store.Post().GetSingle(postId)
```

Comment:
I suppose we don't need a own go routine for this. But leaving the decision to @jespino.

Reply:
In this case we need it because we want to start to get the post and in parallel get the file infos for the post, so... we need to make it explicitly asynchronous.